src/main/java/in/projecteka/consentmanager/clients/ClientError.java

```diff
@@ -134,7 +133,7 @@ public static ClientError consentArtefactNotFound() {
     }
 
     public static ClientError consentArtefactForbidden() {
-        return new ClientError(FORBIDDEN,
+        return new ClientError(INTERNAL_SERVER_ERROR,
                 new ErrorRepresentation(new Error(CONSENT_ARTEFACT_FORBIDDEN, "Cannot retrieve Consent artefact")));
     }
```

Comment:
Why we are changing this?

Reply:
it's mentioned in the doc to change http status from forbidden to internal server